### ISSUE TITLE
ElementNotInteractableException: Message: Element is not visible

### ISSUE THREAD

**Issue by ZeusIssueReporter** [Ekultek/Zeus-Scanner#91]
Zeus version:
`1.0.42.49ab`

Error info:
```Traceback (most recent call last):
  File "/home/user/Desktop/Zeus-Scanner-master/var/google_search/search.py", line 233, in parse_search_results
    query_url = get_urls(query, url_to_search, verbose=verbose, user_agent=user_agent, proxy=proxy_string)
  File "/home/user/Desktop/Zeus-Scanner-master/var/google_search/search.py", line 137, in get_urls
    search.send_keys(query)
  File "/usr/local/lib/python2.7/dist-packages/selenium/webdriver/remote/webelement.py", line 350, in send_keys
    'value': keys_to_typing(value)})
  File "/usr/local/lib/python2.7/dist-packages/selenium/webdriver/remote/webelement.py", line 499, in _execute
    return self._parent.execute(command, params)
  File "/usr/local/lib/python2.7/dist-packages/selenium/webdriver/remote/webdriver.py", line 297, in execute
    self.error_handler.check_response(response)
  File "/usr/local/lib/python2.7/dist-packages/selenium/webdriver/remote/errorhandler.py", line 194, in check_response
    raise exception_class(message, screen, stacktrace)
ElementNotInteractableException: Message: Element is not visible
````

Running details:
`Linux-4.12.0-parrot6-686-pae-i686-with-Parrot-3.8-JollyRoger`

Commands used:
`zeus.py -d manager.php?UID=`

Log file info:
```2017-10-13 11:29:00,734;zeus-log;INFO;log file being saved to '/home/user/Desktop/Zeus-Scanner-master/log/zeus-log-4.log'...
2017-10-13 11:29:00,735;zeus-log;INFO;using default search engine...
2017-10-13 11:29:00,736;zeus-log;INFO;starting dork scan with query 'manager.php?UID='...
2017-10-13 11:29:00,737;zeus-log;INFO;attempting to gather query URL...
2017-10-13 11:29:01,313;zeus-log;INFO;firefox browser display will be hidden while it performs the query...
2017-10-13 11:29:01,314;zeus-log;WARNING;your web browser will be automated in order for Zeus to successfully bypass captchas and API calls. this is done in order to grab the URL from the search and parse the results. please give selenium time to finish it's task...
2017-10-13 11:30:01,276;zeus-log;INFO;browser will open shortly...
2017-10-13 11:30:20,194;zeus-log;INFO;searching 'http://google.com' using query 'manager.php?UID='...
2017-10-13 11:30:21,884;zeus-log;ERROR;[7;31;31msearch.pyc failed to gather the URL from search engine, caught exception 'Message: Element is not visible' exception has been logged to current log file...
Traceback (most recent call last):
  File "/home/user/Desktop/Zeus-Scanner-master/var/google_search/search.py", line 233, in parse_search_results
    query_url = get_urls(query, url_to_search, verbose=verbose, user_agent=user_agent, proxy=proxy_string)
  File "/home/user/Desktop/Zeus-Scanner-master/var/google_search/search.py", line 137, in get_urls
    search.send_keys(query)
  File "/usr/local/lib/python2.7/dist-packages/selenium/webdriver/remote/webelement.py", line 350, in send_keys
    'value': keys_to_typing(value)})
  File "/usr/local/lib/python2.7/dist-packages/selenium/webdriver/remote/webelement.py", line 499, in _execute
    return self._parent.execute(command, params)
  File "/usr/local/lib/python2.7/dist-packages/selenium/webdriver/remote/webdriver.py", line 297, in execute
    self.error_handler.check_response(response)
  File "/usr/local/lib/python2.7/dist-packages/selenium/webdriver/remote/errorhandler.py", line 194, in check_response
    raise exception_class(message, screen, stacktrace)
ElementNotInteractableException: Message: Element is not visible


2017-10-13 11:30:29,666;zeus-log;INFO;[32mZeus got an unexpected error and will automatically create an issue for this error, please wait...[0m
2017-10-13 11:30:29,667;zeus-log;INFO;[32mgetting authorization...[0m
2017-10-13 11:30:29,669;zeus-log;INFO;[32mextracting traceback from log file...[0m
```

**Comment by Ekultek** [comment]
This means that the `div` tags are faked and there’s more JavaScript in the HTML then there is HTML. Found a reference for a fix https://stackoverflow.com/questions/27927964/selenium-element-not-visible-exception

**Comment by Ekultek** [comment]
Patched via bef138e
